api/app/signals/apps/email_integrations/actions/assigned_action.py

```diff
@@ -0,0 +1,48 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2021 - 2022 Gemeente Amsterdam
+import logging
+
+from django.conf import settings
+from django.core.mail import send_mail
+from django.template import Context, Template, loader
+
+from signals.apps.email_integrations.exceptions import URLEncodedCharsFoundInText
+from signals.apps.email_integrations.actions.abstract import AbstractSystemAction
+from signals.apps.email_integrations.models import EmailTemplate
+
+logger = logging.getLogger(__name__)
+
+
+class AssignedAction(AbstractSystemAction):
```

Comment:
Can you add some comments explaining what this action does, and how it is different then all other actions (mailing a civil servant instead of a reporter)?

Reply:
Done 👍